Update dailyMessageQuotaUsed metric description

astaykov opened the pull request:
Adding more explanation to the most confusing value - dailyMessageQuotaUsed. Which is a cumulative value holding the total number of messages used since 00:00 UTC up to the point of time being asked. It cannot be split into time grains (each time grain will hold the cumulative daily value, and not the time slice value).

azurecla commented:
Hi __@setuc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

astaykov committed:
Update dailyMessageQuotaUsed metric description

Adding more explanation to the most confusing value - dailyMessageQuotaUsed. Which is a cumulative value holding the total number of messages used since 00:00 UTC up to the point of time being asked. It cannot be split into time grains (each time grain will hold the cumulative daily value, and not the time slice value).

anirudhcavale reviewed (changes requested):
Hi @astaykov update looks great. Can you just update the following:
"This is cumulative value that is being reset to zero at 00:00 UTC."
to 
"This is a cumulative value that is reset to zero at 00:00 UTC every day."

astaykov committed:
Update monitoring-supported.metrics.md

Polished the text according to feedback from owner.
